Pages/Bookmarks.js

```diff
@@ -59,15 +59,15 @@ if (locationIds !== null) {
 if (isLoading) {
 return (
   <View style={styles.loading}>
-    <Text >Loading...</Text>
+    <Text >You have no bookmarks</Text>
```

Comment:
Is is possible to add another conditional rendering that requires user to log in before he/she can have access to the bookmark page?